README.rst

```diff
@@ -95,7 +95,7 @@ Tests and coverage can be run using Pytest.
 
 .. code-block:: bash
 
-    pytest tests/ --cov=dwave.gate
+    python -m pytest tests/ --cov=dwave.gate
```

Comment:
Changing this so that it uses the same command as in the makefile. This is likelier to run the correct installation of pytest (if the user happens to have several versions), but with the downside (?) of testing the local files rather than the installed package.